src/main/webapp/dev/js/main/instructorCourseEdit.es6

```diff
@@ -136,6 +136,7 @@ function hideTunePermissionDiv(instrNum) {
  */
 function enableFormEditInstructor(number) {
     $(`#instructorTable${number}`).find(':input').not('.immutable').prop('disabled', false);
+    if (!$('.display-to-student').prop('checked')) $('.title-displayed').prop('disabled', true);
```

Comment:
The selector here does not seem to use the number, resulting in it changing the state of every "display to students as" checkbox to `disabled` if the first instructor in the list (as opposed to the instructor that is being edited) is not visible to students.

Also, use
```javascript
if (condition) {
    // statement
}
```
even if there is only one statement for the conditional.